### PR TITLE
github action to create changeset

### PR DESCRIPTION
**Pull request by lukasoppermann**
This PR adds a workflow to create a changeset when a post request is send to this PR.

### CODE REVIEW

**Review by rezrah — APPROVED** [review]
looks great!

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js 18.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18.x
+
+      - name: Install Dependencies
+        run: yarn
```
Comment (rezrah) [inline .github/workflows/release.yml:45]: Can we switch to npm?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/lukasoppermann/figmaWebhook/issues"
+  },
+  "homepage": "https://github.com/lukasoppermann/figmaWebhook#readme",
+  "description": "",
+  "scripts": {
+    "release": "changeset publish",
+    "createChangeset": "npx ts-node --transpile-only .github/actions/create-changeset.ts"
```
Comment (rezrah) [inline package.json:18]: nit: can we use kebab-case for npm commands like we do elsewhere?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lukasoppermann/figmaWebhook.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/lukasoppermann/figmaWebhook/issues"
+  },
+  "homepage": "https://github.com/lukasoppermann/figmaWebhook#readme",
```
Comment (rezrah) [inline package.json:14]: FYI, these should point at primer org now?

Comment (lukasoppermann) [reply package.json:14]: Very good point, will fix.

**Pull request merged**